spec/index.bs

```diff
@@ -8488,6 +8495,13 @@ dictionary GPURenderPipelineDescriptor
             |descriptor|.{{GPURenderPipelineDescriptor/fragment}} is less
             than |device|.limits.{{supported limits/maxInterStageShaderVariables}}.
             (This follows from the above rules.)
+        1. If |device|.{{device/[[features]]}} does not [=list/contain=] {{GPUFeatureName/"core-features-and-limits"}}:
+                <div class="compatmode">
+            - For each user-defined |input| of |descriptor|.{{GPURenderPipelineDescriptor/fragment}}:
+                - If the [=interpolation=] of the |input| is [=interpolation type/linear=], return `false`.
+                - If the [=interpolation=] of the |input| is [=interpolation type/flat=] and the [=interpolation sampling=] is not [=interpolation sampling/either=], return `false`.
+                - If the [=interpolation sampling=] of the |input| is [=interpolation sampling/sample=] or [=interpolation sampling/first=], return `false`.
```

Comment:
maybe factor this out so it doesn't have to be duplicated

Reply:
Sure. Should this be in a `<div algorithm>` and a `<dfn abstract-op>`? I'm at the limits of my bikeshed knowledge here.